<launch>

	<arg name="mission_index" default="1" doc="index of the desired mission to run" />

	<node pkg="aa241x_mission" type="aa241x_mission_node" name="aa241x_mission_node" output="screen">
		<param name="mission_index" value="$(arg mission_index)" />
		<param name="mission_file" value="$(find aa241x_mission)/missions/id$(arg mission_index).mission" type="str" />
	</node>

</launch>